pom.xml

```diff
@@ -205,7 +205,7 @@
         <maven.compiler.release>11</maven.compiler.release>
 
         <!-- maven-enforcer-plugin -->
-        <min-maven-version>3.8.1</min-maven-version>
+        <min-maven-version>3.8.2</min-maven-version>
```

Comment:
Maybe run the build in docs module because this property is an input into `antora.yaml`:

https://github.com/apache/camel-quarkus/blob/main/docs/antora.yml#L30